### DISCUSSION TITLE
[AMD] Vsync seems to be forced in osu! lazer

### DISCUSSION THREAD

**Discussion by grigory-bogush** [ppy/osu#27650] [Q&A] [answered]
### Type

Performance

### Bug description

It would appear that on AMD (in my case on RX 7800XT) the osu! lazer client seems to be forced to vsync to dispaly refresh rate regardless of what I do. The gameplay is also very choppy (so even if it runs at 75 it feels really choppy so this 'vsync' is probably not properly aligned with freesync on my display).

Old osu! client runs fine in 240+ fps fine though. There is also some weird behavior with amd control panel where it won't let me turn off the enchanced sync option for some reason. I have not been able to figure out what is up with that so help would be appricated.

Actual for both OpenGL and DirectX 11 renderer.

### Screenshots or videos

https://github.com/ppy/osu/assets/23219057/f6ef9fec-67f4-4b81-8f29-fa48538dcacf

![image](https://github.com/ppy/osu/assets/23219057/f1a8f712-9b28-4b13-bf1a-9eceb78c1e79)

https://github.com/ppy/osu/assets/23219057/28f0fb51-06f8-42c6-a1f9-9b8d61556239

### Version

2024.312.1-lazer

### Logs

This repeats always so this is just some logs from one of the tries:

[compressed-logs.zip](https://github.com/ppy/osu/files/14627205/compressed-logs.zip)

**Comment by peppy** [comment, marked as answer]
Probably means you have gsync turned on or something.

The game does not force vsync. Your system is.

**Reply by grigory-bogush** [reply]
The issue is, it doesn't happen for the older version. Maybe an issue with AMD soft not detecting osu! lazer vs original osu! properly or carrying over settings incorrectly...?

No idea how to best narrow something like this down

**Reply by grigory-bogush** [reply]
UPD: I think the profile for amd for both clients seems to be the same and it seems to break lazer! for some reason.
I sent a bug report to AMD, hopefully this gets fixed
